### DISCUSSION TITLE
Database schema diagnostic

### DISCUSSION THREAD

**Discussion by Hennard** [MISP/MISP#8071] [General]
Hi, is this normal:
Warning The MISP database state does not match the expected schema. Resolving these issues is recommended.

Column `comment` is different | comment NO text 65535 utf8mb4_unicode_ci text | comment YES text 65535 utf8mb4_unicode_ci text
Column `comment` is different | comment YES text 65535 utf8_unicode_ci text | comment NO text 65535 utf8_unicode_ci text
Column `comment` is different | comment NO text 65535 utf8_unicode_ci text | comment YES text 65535 utf8_unicode_ci text

Get this on the VMware setup: MISP_v2.4.152@74cdc9d-VMware.zip
Currently installed version… v2.4.152

**Comment by Giblin91** [comment]
I have same issue, but with self deployed MISP instance. Actually I have many more warning, like for all integers columns:
Description | Expected schema | Actual schema
Column `id` is different | id NO int 10 **int(11)** auto_increment | id NO int 10 **int** auto_increment

Not sure why or how to approach/handle this
